schemas/settings.json

```diff
@@ -0,0 +1,144 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "definitions": {
+        "color": {
+            "type": "string",
+            "enum": [
```

Comment:
I am not even sure that this setting should have a whitelist.
The user can create any color function they want, and then place its name here.

Reply:
What color formats are permitted? RBG maybe too?

Reply:
No. Any string is allowed here, and the user is expected to have a function with the same name.
The colors you outlined in the list, are just the colors that the [built in color library](https://github.com/DannyBen/bashly/blob/master/lib/bashly/libraries/colors/colors.sh) brings.

Users don't have to use it, they can have a function named `command_color` and this will be the string that they will use in the colors in settings.

Reply:
Now any color is permitted, but standard ones are suggested.

Reply:
Perfect